src/esp/bindings/GfxBindings.cpp

```diff
@@ -46,7 +46,7 @@ void initGfxBindings(py::module& m) {
           R"(RenderCamera: The object of this class is a camera attached
       to the scene node for rendering.)");
 
-  py::enum_<RenderCamera::Flag> flags{render_camera, "Flags", "Flags"};
+  py::enum_<RenderCamera::Flag> flags{m, "Flags", "Flags"};
```

Comment:
This should still go on the render camera IMO.  You can access it from python as `Camera.Flags`